src/OrchardCore.Modules/OrchardCore.Flows/Views/FlowMetadata.Edit.cshtml

```diff
@@ -18,22 +18,22 @@
 
 <div class="btn-group btn-group-sm" data-toggle="buttons" role="group">
     <label class="btn btn-secondary @(Model.Size == 25 ? "active" : null)">
-        <input asp-for="Size" type="radio" autocomplete="off" value="25">@T["25%"]
+        <input asp-for="Size" type="radio" autocomplete="off" value="25">25%
```

Comment:
Maybe keep because we never know if a language wants to use custom `%` symbol, or needs a space, or RTL, ...

Reply:
I don't think it's useful because % is same for all the cultures even RTL, furthermore the size here in percentage unit